tests/performance/test-do-pings.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Automattic\VIP\Performance;
+
+class Do_Pings_Test extends \WP_UnitTestCase {
+	public function setUp() {
+		parent::setUp();
+	}
+
+	public function tearDown() {
+		parent::tearDown();
+	}
+
+	public function test__block_encloseme_metadata_filter_should_respect_update_value_if_not_encloseme() {
+		$should_update = true;
+		$object_id = 1;
+		$meta_key = 'test';
+		$meta_value = 'random value';
+		$unique = true;
+
+		$result = block_encloseme_metadata_filter( $should_update, $object_id, $meta_key, $meta_value, $unique );
+		
+		$this->assertTrue( $result, 'block_encloseme_metadata_filter should return true' );
+		$this->assertTrue( \apply_filters( 'add_post_metadata', $should_update, $object_id, $meta_key, $meta_value, $unique ), 'add_post_metadata should return true' );
+
+		$should_update = false;
+
+		$result = block_encloseme_metadata_filter( $should_update, $object_id, $meta_key, $meta_value, $unique );
+
+		$this->assertFalse( $result, 'block_encloseme_metadata_filter should return false' );
+		$this->assertFalse( \apply_filters( 'add_post_metadata', $should_update, $object_id, $meta_key, $meta_value, $unique ), 'add_post_metadata should return false' );
```

Comment:
I would suggest making this more of an integration-y test where we call `add_post_meta` and verify it works or not works (depending on the meta key). I think, in theory, the unit test makes sense, but what's more interesting and useful is whether it actually gets applied in practice (i.e. via add/update post meta).

Thoughts?

Reply:
Sounds good to me.

Reply:
Added the new tests in https://github.com/Automattic/vip-go-mu-plugins/pull/1706/commits/d857a662fc4e6d8d4c5e87aa73385f8aef5f38fb